misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -46,67 +47,55 @@ abstract class PathAssertionsSpec(
                     )
                 }
             }
-        }
-        context("existing file") {
-            it("does not throw") {
+
+            it("${exists.name} does not throw when file exists") {
```

Comment:
once you have two contexts you can simplify this to 
```suggestion
            it("${exists.name} does not throw") {
```
Same for folder below and same for the other `it` in the `file does not exist` and `folder does not exist`

Reply:
Fixed that and other related comments, thanks!

I added the additional description because I thought it would be helpful for test reporting

Now the test report looks like this and I think it's not clear which context the failing test belongs to:

![image](https://user-images.githubusercontent.com/2130907/66644206-a2720200-ec20-11e9-957b-e5b1164e5de9.png)

Reply:
Yeah... well, it does not include the hierarchy as intellij does
![image](https://user-images.githubusercontent.com/5557885/66647988-a7d44a00-ec2a-11e9-8dfa-08b050595f15.png)
Maybe there is a config setting so that this is reflected as well. 
I think it will already be better if you change the `it` and include the path as well

Reply:
I found this: https://github.com/gradle/gradle/issues/5273
Let's ignore this for know, I guess gradle will fix this and include the hierarchy in the future

Reply:
I see. Sounds good. It should get fixed in the future. I'll address the other comments later in the day. Thanks.